src/main/kotlin/uk/gov/dluhc/printapi/PrintApiApplication.kt

```diff
@@ -3,11 +3,15 @@ package uk.gov.dluhc.printapi
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.context.properties.ConfigurationPropertiesScan
 import org.springframework.boot.runApplication
+import org.springframework.integration.annotation.IntegrationComponentScan
+import org.springframework.integration.config.EnableIntegration
 
 /**
  * Spring Boot application bootstrapping class.
  */
 @SpringBootApplication
+@IntegrationComponentScan
+@EnableIntegration
```

Comment:
What do these 2 annotations do? (feels like I should know, but clearly I don't! 😬 🤣 )

Reply:
Ah, enables Spring Integration?
(that would be why I dont know about them - SI is voodoo! 😁 🤣 )

Reply:
Spring Integration makes working with the SFTP server far easier than working directly with the JSCH library 👍